public/app/features/provisioning/ConfigFormGithubCollapse.tsx

```diff
@@ -0,0 +1,40 @@
+import { config } from '@grafana/runtime';
+import { Alert, ControlledCollapse, Field } from '@grafana/ui';
+
+import { checkPublicAccess } from './Setup/utils';
+
+export interface ConfigFormGithubCollpaseProps {
+  previews: React.ReactElement;
+}
+export function ConfigFormGithubCollpase({ previews }: ConfigFormGithubCollpaseProps) {
+  return (
+    <ControlledCollapse label="Realtime support" isOpen={true}>
+      {!checkPublicAccess() && <Alert title={'Github webhooks require a public URL'} severity={'warning'} />}
```

Comment:
@MissingRoberto -- can you push any updates that would open the setup modal?

I'm not sure how much or where we want to refactor that

Reply:
Updated the PR so we have explicit inline errors for github:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/19ecf775-93fc-43f2-9b01-c4e2ab29693f" />

Style will need some ❤️ from patric++, but it seems good to indicate which features won't work near where you try to configure them